content/swiftui/concepts/viewmodifier/terms/baselineOffset/baselineOffset.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.baselineOffset()'
+Description: 'Moves text up or down relative to its baseline.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.baselineOffset()`** modifier method moves text in a view [view](https://www.codecademy.com/resources/docs/swiftui/views) vertically relative to it's baseline. The bounds of the text's parent view expand to contain the modified text.
+
+## Syntax
+
+```pseudo
+Text("This text is shifted up.")
+    .baselineOffset(10)
+Text("This text is shifted down.")
+    .baselineOffset(-10)
+```
+
+The `.baselineOffset()` modifier method uses the following parameter:
+
+- `baselineOffset` specifies how many points the text is shifted up or down. The following options are possible:
```

Comment:
This line and the following table is misleading, implying that 0,5,-5 are the only valid options. Simply say the parameter can be positive, negative or zero.